src/main/java/org/folio/dew/config/JacksonConfiguration.java

```diff
@@ -101,13 +105,27 @@ public JobParameter<?> deserialize(JsonParser jp, DeserializationContext ctxt) t
           default -> node.asText();
         };
 
+        if (identifying){
+          return checkForExisted(value, clazz, identifying);
+        }
+
         return new JobParameter<>(value, clazz, identifying);
       } catch (ClassNotFoundException e) {
         throw new RuntimeException("Cannot create Job parameter with the class " + type, e);
       } catch (ParseException e) {
         throw new RuntimeException("Invalid Date format: " + node.asText(),  e);
       }
     }
+
```

Comment:
It looks like workaround. Please, add brief comment to explain what happens here.